### PR TITLE
Planet 5466 - New Accordion block

### PR DESCRIPTION
**Pull request by mleray**
See https://jira.greenpeace.org/browse/PLANET-5466

You can test the block [here](https://www-dev.greenpeace.org/test-phoebe/accordion-test/)

### CODE REVIEW

File: assets/src/blocks/Accordion/AccordionBlock.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      icon: 'menu',
+      category: 'planet4-blocks-beta',
+      keywords: [
+        __(BLOCK_NAME),
+        __('faq'),
+        __('collapsible')
+      ],
+      attributes,
+      edit: AccordionEditor,
+      save: frontendRendered(BLOCK_NAME),
```
Comment (Inwerpsel) [inline assets/src/blocks/Accordion/AccordionBlock.js:39]: Do we need to render this on the front end? Can we instead save the html?

Comment (mleray) [reply assets/src/blocks/Accordion/AccordionBlock.js:39]: You mean using directly `AccordionFrontend` here?

File: assets/src/blocks/Accordion/AccordionBlock.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      tabs: {
+        type: 'array',
+        default: [],
+      },
+    };
+
+    const styles = [
+      {
+        name: 'dark',
+        label: __( 'Dark', 'planet4-blocks' ),
```
Comment (sagarsdeshmukh) [inline assets/src/blocks/Accordion/AccordionBlock.js:31]: Here the textdomain will be `planet4-blocks-backend`

**Comment by sagarsdeshmukh** [comment]
The new Accordion blocks looks good 👍 
Following are some findings, while testing the block on local -
- On change of accordion style , and refresh the page - got “This block contains unexpected or invalid content.” Error.
- Added a accordion block - add accordion tab 1 with button option publish the post, refresh the page - block breaks on backend
- Side note - The post picks accordion block details in excerpt if its a first block in post.
- The “Edit as HTML” option breaks the block on backend, better to remove it for this block.

**Comment by mleray** [comment]
@sagarsdeshmukh Great findings, thanks! I've pushed fixes, can you please re-check? 🙂

**Review by Inwerpsel — COMMENTED** [review]
Just noticed I hadn't submitted these comments yet from a few days back, will look further into it later :slightly_smiling_face:

**Comment by sagarsdeshmukh** [comment]
> @sagarsdeshmukh Great findings, thanks! I've pushed fixes, can you please re-check? 🙂

Hi @mleray , The previous issues are fixed. Functionality wise PR looks good to me 👍 
One small thing, if editor add bold text in accordion tab heading on post, it will break the tab structure on frontend.

**Comment by comzeradd** [comment]
@mleray can you also rebase with master branch, so the acceptance tests pass?

**Comment by mleray** [comment]
> One small thing, if editor add bold text in accordion tab heading on post, it will break the tab structure on frontend.

Once again, good catch @sagarsdeshmukh 😄 After discussing it with Magali, I've removed the formatting controls on all RichText components in AccordionEditor using `formattingControls={[]}`. I've also opened a [ticket](https://jira.greenpeace.org/browse/PLANET-5780) to do the same in other WYSIWYG blocks since it's probably causing issues there too!

**Review by sagarsdeshmukh — APPROVED** [review]
Nice Work @mleray 👍

**Pull request merged**